### ISSUE TITLE
Allow any sender

### ISSUE THREAD

**Issue by cao-** [tranquvis/SimpleSmsRemote#14]
Instead of relying only on the list of granted phones, we could allow anyone to send commands through messages.  This can be achieved by allowing the user of the application to set a password in the application itself.  I explain it by an example.  Suppose the password I set is "cao".  If I want to get my phone location, I can send
rc get location
using a granted phone, or I can send something like
rc cao get location
using any phone.
This should be secure, because only the user of the application knows the password.

This could be useful in situations like:
-->  The owner of the granted phone changed number and I forgot to update the list of granted phones in the application
-->  I lost my phone and I am alone.  I can't contact any of the owners of the granted phones because I don't have a phone.  But I can ask help to a random stranger.

**Comment by lameventanas** [comment]
I would rephrase this as: implement password based authentication for remote commands.

**Comment by tranquvis** [comment]
I think the use cases described here can be done with the new feature/module "Grant Phone Remotely".
